locations/spiders/nobis_printen_de.py

```
@@ -0,0 +1,52 @@
+from scrapy.http import Response
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.google_url import url_to_coords
+from locations.hours import CLOSED_DE, DAYS_DE, OpeningHours
+from locations.items import Feature
+
+
+class NobisPrintenDESpider(SitemapSpider):
+    name = "nobis_printen_de"
+    item_attributes = {"brand": "Nobis", "brand_wikidata": "Q1994827", "state": "DE-NW"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)